helpers/router.js

```diff
@@ -20,18 +20,35 @@ var Router = function () {
 		var router = this;
 
 		Object.keys(config).forEach(function (params) {
+
```

Comment:
so as the routes structure didn't change you can just revert `router.map` function to the previous state